Config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@targets_export_name@.cmake")
+check_required_components(glad)
```

Comment:
This file is the template for the glad's CMake package configuarion file. It is used to generate gladConfig.cmake, which is installed to <prefix>/lib/cmake/glad. Having this file installed (along with the other 'targets' and 'versions' cmake files) allows the package to be used with CMake's newer package configuration system. No more writing 'FindXXX.cmake' files by hand! 
Using the newer package configuration system, glad can be included in a project just by installing it to a prefix and including the following lines of CMake:
```
find_package(glad CONFIG REQUIRED)
...
target_link_libraries(target glad::glad)
# Note: include directories are also imported by the above line, just by using the glad target.
```
(Note that "glad::" is the value namespace variable defined in the CMakeLists.txt file)